tests/test_analytical_fit.py

```diff
@@ -187,13 +188,13 @@ def test_weights(self):
         fit_result = fit.execute()
 
         self.assertAlmostEqual(num_result.value(sqrt_g_inv), fit_result.value(sqrt_g_inv))
-        self.assertAlmostEqual(num_result.value(b), fit_result.value(b))
+        self.assertAlmostEqual(num_result.value(b) / fit_result.value(b), 1.0, 5)
```

Comment:
Elegant. I like it. Maybe set the precision (5) as a global variable for all the tests?

Reply:
Yes, relative testing everywhere is probably the way to go.